tools/UpdatePaths/Scripts/70773_toggleable_helmets_repath.txt

```diff
@@ -0,0 +1,4 @@
+#comment This repaths the riothelmet and justice helmet to toggleable subtypes. see (https://github.com/tgstation/tgstation/pull/70773)
+
+/obj/item/clothing/head/helmet/riot : /obj/item/clothing/head/helmet/toggleable/riot{@OLD}
+/obj/item/clothing/head/helmet/justice : /obj/item/clothing/head/helmet/toggleable/justice{@OLD}
```

Comment:
can you include subtypes here since there's ``/justice/escape``

Reply:
![image](https://user-images.githubusercontent.com/6209658/198043245-c9471dca-967a-4e0b-90a2-8749a51bc5cd.png)
there documentation for that anywhere????